service/src/test/java/com/ge/predix/controller/test/SubjectPrivilegeManagementControllerIT.java

```diff
@@ -378,6 +379,30 @@ public void testPOSTCreateSubjectThenUpdateAttributes() throws Exception {
     }
 
     @Test
+    public void testSubjectWithParents() throws Exception {
+        if (XFiles.titanProfileActive()) {
+            return;
+        }
+
+        BaseSubject subject = this.jsonUtil.deserializeFromFile(
+            "controller-test/a-subject-with-parents.json", BaseSubject.class);
+        Assert.assertNotNull(subject);
+
+        // create subject in first zone
+        MockMvcContext putContext = this.testUtils.createWACWithCustomPUTRequestBuilder(
+            this.wac, this.testZone.getSubdomain(), SUBJECT_BASE_URL + "/dave-with-parents");
+        final MvcResult result = putContext.getMockMvc()
+                                           .perform(putContext.getBuilder().contentType(MediaType.APPLICATION_JSON)
+                                                              .content(OBJECT_MAPPER.writeValueAsString(subject)))
+                                           .andExpect(status().isNotImplemented())
+                                           .andReturn();
+
+        Assert.assertEquals(result.getResponse().getContentAsString(),
+                            "{\"ErrorDetails\":{\"errorCode\":\"FAILURE\","
```

Comment:
I recommend we use BaseRestAPI.PARENTS_ATTR_NOT_SUPPORTED_MSG instead of harcoding the message in line 402.